swagger.yml

```diff
@@ -2613,6 +2613,28 @@ components:
           x-go-name: Events
       type: object
       x-go-package: github.com/TykTechnologies/tyk/apidef
+    RateLimitMeta:
+      properties:
+        disabled:
+          type: boolean
+          x-go-name: Disabled
+        method:
+          type: string
+          x-go-name: Method
+        path:
+          type: string
+          x-go-name: Path
+        per:
+          type: number
+          format: double
+          x-go-name: Per
+        rate:
+          type: number
+          format: double
+          x-go-name: Rate
+      title: RateLimitMeta configures rate limits per API path.
```

Comment:
```suggestion
      title: RateLimitMeta configures rate limits per API endpoint.
```

Reply:
Should this be the general case (i.e. "configures rate limits per API endpoint") or more specific (i.e. "configures the rate limit for an API endpoint")?

Reply:
So this is classic apidef, here it's essentially by path as it doesn't have an endpoint concept. In OAS we link it to endpoints currently, both is correct otherwise. Since it's classic docs, i would leave path.

Reply:
OK, thanks - please feel free to decline this suggestion. :)